docs/changelog.md

```diff
@@ -12,6 +12,7 @@
 #### Changed
 * Upgraded to Angular 8 (including compatible TypeScript / CLI versions)
 * Improved Notification logs
+* Improved audit log page
```

Comment:
* Improved Notification and Audit Logs

In Added please add, 
* Filtering by service and task in System logs page, and local search facility